.github/workflows/changed_file_trigger_examples_check.yml

```diff
@@ -0,0 +1,117 @@
+name: change_examples_check_all
+run-name: ${{ github.actor }} is testing out GitHub Actions 🚀
+on:
+  pull_request:
+
+  schedule:
+    # run at 00:00 of every Sunday
```

Comment:
GitHub action does not use Singapore time. Change this cron expression to Singapore time.